src/index.html

```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a href="#apple" class="link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#smartphones" class="link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a href="#laptops&computers"
+              class="link"
+              data-qa="hover"
+            >Laptops & Computers</a>
+          </li>
+          <li class="nav_item">
+            <a href="#gadgets" class="link">Gadgets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#tablets" class="link">Tablets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#photo" class="link">Photo</a>
+          </li>
+          <li>
```

Comment:
```suggestion
          <li class="nav_item">
```

Reply:
I've left last <li> without class to avoid applying right margin according Checklist p.12
12. [TESTS] - Check the [mockup](https://www.figma.com/file/KAV1NnDp7hgQtPnaD6XdOcnG/Moyo-Header?node-id=0%3A1)
again and see the distance between `О` in `VIDEO` to right screen side. Link has
no margin on the right.

Please advise if class has to be set for last link?

Reply:
Don't  reply,   I found resolve in next your comment